src/pages/workspaces/workspace/tools/LaunchAnalysisModal.js

```diff
@@ -64,12 +66,14 @@ export default class LaunchAnalysisModal extends Component {
     const { workspaceId: { namespace, name } } = this.props
     const { entityType } = this.state
 
-    Rawls.workspace(namespace, name).entityMetadata().then(
+    !!entityType && Rawls.workspace(namespace, name).entityMetadata().then(
       entityMetadata => this.setState({ entityMetadata }),
       attributeFailure => this.setState({ attributeFailure })
     )
 
-    this.loadEntitiesOfType(entityType)
+    !!entityType && this.loadEntitiesOfType(entityType)
+
+    _.isUndefined(entityType) && this.launch()
```

Comment:
This looks like it will launch immediately with no confirmation click. Is that what we want?

Also, if this is an either/or, perhaps a ternary would be clearer.